examples/acoustics/acoustic.py

```diff
@@ -0,0 +1,179 @@
+
+import os
+import argparse
+
+import numpy as np
+import pandas as pd
+from matplotlib import pyplot as plt
+import matplotlib.ticker as ticker
+
+import flow360client
+
+def getCaseConfigParameters(caseId):
+    return flow360client.case.GetCaseInfo(caseId)['runtimeParams']
+
+def readAcousticCSVFile(acsv):
+    # read acoustic csv when the file is available via python api
+    # with tempfile.TemporaryDirectory() as tmpdir:
+    #     tmpfile = os.path.join(tmpdir, 'aeroacoustics_v3.csv')
+    #     flow360client.case.DownloadFile(caseId, 'aeroacoustics_v3.csv', tmpfile)
+    pressure_data = pd.read_csv(acsv,skipinitialspace=True)
+    times = pressure_data['physical_time']
+    steps = pressure_data['physical_step']
```

Comment:
There are going to be a few changes in the release version of aeroacoustics, physical_time will be renamed to time. Will let you know when the final release version is ready for testing

Reply:
OK. Looking forward.

Reply:
File will be renamed to total_acoustics_v3.csv for release-23.2.3.0